packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -47,7 +65,7 @@ export class AzureClient {
             package: "no-dynamic-package",
             config: {},
         });
-        await container.attach({ url: containerConfig.id });
+        this.containerList[containerConfig.id] = container;
```

Comment:
I'd suggest refraining from storing the list of detached containers in the client since while its in a detached state, it doesn't have any specific relation to the service. Perhaps there can be an `attach` call on the FluidContainer itself that takes in the ID and the service-specific client that will be doing the attaching?